### ISSUE TITLE
Null object tried to be parsed as JSON

### ISSUE THREAD

**Issue by eladkarako** [TeamNewPipe/NewPipe#3237]
## Exception
* __User Action:__ requested channel
* __Request:__ https://www.youtube.com/channel/UCRedgf5rdWzUCIP5HGwU84w
* __Content Language:__ GB
* __Service:__ YouTube
* __Version:__ 0.18.6
* __OS:__ Linux Android 9 - 28

<details><summary><b>Crash log</b></summary><p>

```
java.lang.NullPointerException: Attempt to invoke virtual method 'com.grack.nanojson.JsonObject com.grack.nanojson.JsonObject.getObject(java.lang.String)' on a null object reference
	at org.schabi.newpipe.extractor.services.youtube.extractors.YoutubeChannelExtractor.getPage(YoutubeChannelExtractor.java:200)
	at org.schabi.newpipe.extractor.channel.ChannelInfo.getMoreItems(ChannelInfo.java:53)
	at org.schabi.newpipe.util.ExtractorHelper.lambda$getMoreChannelItems$5(ExtractorHelper.java:131)
	at org.schabi.newpipe.util.-$$Lambda$ExtractorHelper$W_-kT4Q2PcaBmHMRES2jl1YH31Y.call(Unknown Source:6)
	at io.reactivex.internal.operators.single.SingleFromCallable.subscribeActual(SingleFromCallable.java:44)
	at io.reactivex.Single.subscribe(Single.java:3438)
	at io.reactivex.internal.operators.single.SingleSubscribeOn$SubscribeOnObserver.run(SingleSubscribeOn.java:89)
	at io.reactivex.Scheduler$DisposeTask.run(Scheduler.java:578)
	at io.reactivex.internal.schedulers.ScheduledRunnable.run(ScheduledRunnable.java:66)
	at io.reactivex.internal.schedulers.ScheduledRunnable.call(ScheduledRunnable.java:57)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:301)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:764)

```
</p></details>
<hr>
While browsing 'whats new' section, headed to the last video in my channel, then clicked my channel to browse all videos, then scrolled up and down, while browsing down (I guess this time the list gets populated) there was an error, which I've pasted (formatted) above.
thanks.

**Comment by wb9688** [comment]
Already fixed in my tabs PR.

**Comment by Stypox** [comment]
@wb9688 could you open a separate PR? I don't think the tabs pr will be part of 0.19.0, because one big PR is already part of it (not sure though ;-) ).

**Comment by TheAssassin** [comment]
You seem to have blocked Team NewPipe. Therefore, we will close this issue now. If you unblock us, we can reopen the issue.
